src/test/java/api/loans/OverrideCheckOutByBarcodeTests.java

```diff
@@ -0,0 +1,267 @@
+
+import static api.support.builders.ItemBuilder.CHECKED_OUT;
+import static api.support.matchers.ItemStatusCodeMatcher.hasItemStatus;
+import static api.support.matchers.TextDateTimeMatcher.isEquivalentTo;
+import static api.support.matchers.ValidationErrorMatchers.hasErrorWith;
+import static api.support.matchers.ValidationErrorMatchers.hasMessage;
+import static api.support.matchers.ValidationErrorMatchers.hasParameter;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.builders.LoanPolicyBuilder;
+import api.support.builders.OverrideCheckOutByBarcodeRequestBuilder;
+import api.support.matchers.UUIDMatcher;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class OverrideCheckOutByBarcodeTests extends APITests {
+
+  private static final DateTime TEST_LOAN_DATE =
+    new DateTime(2019, 4, 10, 11, 35, 48, DateTimeZone.UTC);
+  private static final DateTime TEST_DUE_DATE =
+    new DateTime(2019, 4, 20, 11, 30, 0, DateTimeZone.UTC);
+  private static final String TEST_COMMENT = "Some comment";
+
+  @Test
+  public void canOverrideCheckoutWhenItemIsNotLoanable()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    IndividualResource steve = usersFixture.steve();
+    final UUID checkoutServicePointId = UUID.randomUUID();
+
+    setNotLoanablePolicy();
+    IndividualResource response = loansFixture.overrideCheckOutByBarcode(
+      new OverrideCheckOutByBarcodeRequestBuilder()
+        .forItem(smallAngryPlanet)
+        .to(steve)
+        .at(checkoutServicePointId)
+        .on(TEST_LOAN_DATE)
+        .withDueDate(TEST_DUE_DATE)
+        .withComment(TEST_COMMENT)
+    );
+
+    final JsonObject loan = response.getJson();
+
+    assertThat("action should be checkedoutThroughOverride",
+      loan.getString("action"), is("checkedoutThroughOverride"));
+
+    assertThat("due date should equal to due date from request",
+      loan.getString("dueDate"), isEquivalentTo(TEST_DUE_DATE));
+
+    assertThat("due date should equal to due date from request",
+      loan.getString("actionComment"), is(TEST_COMMENT));
+
+    assertThat(loan.getString("id"), is(notNullValue()));
+
+    assertThat("user ID should match barcode",
+      loan.getString("userId"), UUIDMatcher.is(steve.getId()));
```

Comment:
Personally, I might use a static import here to improve readability of the assertion 

(similarly, the next assert below as well)

Reply:
Done